free-3.md

```diff
@@ -10,41 +10,108 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+My idea will help people organize the tasks they need to do and create a 
+concrete plan of action that they can take. It helps people who struggle with 
+productivity but might be overwhelmed on how to overcome it. There are many
+productivity apps and task managers, but this language would allow someone
+to input their tasks and randomly schedule the tasks into a daily calendar
+based on the length, priority, and frequency of the task. Having this 
+definite scheduling puts more pressure on the user to complete the tasks and
+allow them to spend their time more efficiently.
+
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+A DSL would allow the user to specify the nature of the tasks they need to
+accomplish and be able to type out these tasks more efficiently. A GUI might be
+easier to navigate initially, but having the user to input tasks through code
+would allow them to be more specific with the conditions and parameters associated
+with each task. The computer can then randomly generate an hourly schedule filled 
+with the tasks and gives the user a visual output that outlines the schedule.
+
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+The idea excites me because I know that nearly all of my friends in college have
+struggled with procrastination. I don't think people need to or should be productive
+all the time, but when you really need motivation to be productive, it's nice to
+have a really clear plan of action. Coming up with this action plan is difficult
+and time-consuming to do on your own, so having a language to express the tasks you
+need to accomplish and having a schedule computed for you could be an effective tool.
+I came up with this idea after realizing how much time I spend adding things to my
+task manager and then scheduling tasks in my calendar which sort of negates the
```

Comment:
It seems like you're basically looking to make a constraint solver. Given task X, that will take N hours, and needs to be done by time T, place it into my schedule.